pkg/apis/apps/v1alpha1/statefulset_types.go

```diff
@@ -158,6 +158,10 @@ type StatefulSetSpec struct {
 	// consists of all revisions not represented by a currently applied
 	// StatefulSetSpec version. The default value is 10.
 	RevisionHistoryLimit *int32 `json:"revisionHistoryLimit,omitempty"`
+
+	// paused indicates that the StatefulSet is paused.
+	// +optional
+	Paused bool `json:"paused,omitempty"`
```

Comment:
Should we put `Paused` into `StatefulSetSpec` like `DeploymentSpec`, or into `rollingUpdate` strategy?

Reply:
I prefer to putting Paused into RollingUpdateStatefulSetStrategy since it affects upgrade only (not scaling).

Reply:
already put Paused into RollingUpdateStatefulSetStrategy